test/e2e/transformer/test_transformer_odh.py

```
@@ -0,0 +1,527 @@
+        # Pre-check: request without token should be rejected
+        logger.info("Testing request WITHOUT token (should fail)")
+        try:
+            await predict_isvc(
+                rest_v1_client,
+                service_name,
+                "./data/transformer.json",
+                model_name="mnist",
+                network_layer=network_layer,
+            )
+            pytest.fail("Expected request without token to be rejected")
+        except Exception as auth_err:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)